data/types/blood/glucose/glucose.go

```diff
@@ -33,3 +36,75 @@ func (g *Glucose) Normalize(normalizer data.Normalizer) {
 		g.Value = dataBloodGlucose.NormalizeValueForUnits(g.Value, units)
 	}
 }
+
+func roundToEvenWithDecimalPlaces(v float64, decimals int) float64 {
+	if decimals < 1 {
+		return math.RoundToEven(v)
+	}
+	coef := math.Pow(10, float64(decimals))
+	return math.RoundToEven(v*coef) / coef
+}
+
+type Classification string
+
+const (
+	ClassificationInvalid Classification = "invalid"
```

Comment:
Can we get rid of this edge case?

Reply:
Nope. Go's type system isn't expressive enough.